machine/fsl_t2080rdb/rootconf/sysroot-lib-onie/init-platform

```diff
@@ -0,0 +1,28 @@
+# -*- shell-script -*-
```

Comment:
There is no need to include this file at all as it does nothing.  The default behaviour will be fine.